source/features/bugs-tab.tsx

```diff
@@ -93,7 +93,9 @@ async function init(): Promise<void | false> {
 
 	// Update bugs count
 	try {
-		bugsCounter.textContent = numberFormatter.format(await countPromise);
+		bugsCounter.textContent = abbreviateNumber(await countPromise, 1, {padding: false})
+			// Matches GitHub lowercase `k` style
+			.toLowerCase();
```

Comment:
```suggestion
		bugsCounter.textContent = abbreviateNumber(await countPromise, 1, {
			padding: false,
			symbols: ['', 'k']
		});
```

Reply:
Nevermind

Reply:
```suggestion
		bugsCounter.textContent = abbreviateNumber(await countPromise, 1, {padding: false}).toLowerCase();
```

Reply:
DUHH I was fighting with the update for about 35 min before.......

Reply:
I can imagine 😅